plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -195,6 +196,15 @@ private static void createPomXml(IProject project, AppEngineProjectConfig config
     properties.put("projectGroupId", config.getMavenGroupId()); //$NON-NLS-1$
     properties.put("projectArtifactId", config.getMavenArtifactId()); //$NON-NLS-1$
     properties.put("projectVersion", config.getMavenVersion()); //$NON-NLS-1$
+    
+    String mavenPluginVersion = ArtifactRetriever.DEFAULT.getLatestReleaseVersion(
+        "com.google.cloud.tools", "appengine-maven-plugin") //$NON-NLS-1$ $NON-NLS-2$
+        .toString(); 
+    if (mavenPluginVersion == null) {
+      mavenPluginVersion = "1.3.2";  //$NON-NLS-1$
```

Comment:
Should we make `getLatestReleaseVersion()` take a fallback version?

Reply:
Possibly, though personally I've always found that pattern sort of confusing. I don't usually expect a method argument to be unused or conditionally used.